### DISCUSSION TITLE
Laravel 10

### DISCUSSION THREAD

**Discussion by thisisud** [laravel/framework#49295] [Q&A]
I am a beginner and I am making a project. I have checked all of my is code is correct but in local host it is showing control error. While I make a control of the relevant route. How do I resolve this issue. I've seen many tutorials on Youtube but didn't resolve this issue. Help?

**Comment by shaedrich** [comment]
Could you please provide the error message and the respective code snippet? Furthermore, a category other than "Ideas" might be more suitable 🤔
